docs/datetime.md

```diff
@@ -30,36 +140,308 @@
 
 ### datetime.**nanosecond**: number
 
+The above read-only properties allow accessing each component of the date or time individually.
+
+Usage examples:
+```javascript
+dt = new Temporal.DateTime(1995, 12, 7, 3, 24, 30, 0, 3, 500);
+dt.year         // => 1995
+dt.month        // => 12
+dt.day          // => 7
+dt.hour         // => 3
+dt.minute       // => 24
+dt.second       // => 30
+dt.millisecond  // => 0
+dt.microsecond  // => 3
+dt.nanosecond   // => 500
+```
+
 ### datetime.**dayOfWeek** : number
 
+The `dayOfWeek` read-only property gives the weekday number that the date falls on.
+The weekday number is a value between 1 and 7, inclusive: Monday is 1, and Sunday is 7.
+
+Usage example:
+```javascript
+dt = new Temporal.DateTime(1995, 12, 7, 3, 24, 30, 0, 3, 500);
+['MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT', 'SUN'][dt.dayOfWeek - 1]  // => THU
+```
+
+### datetime.**dayOfYear** : number
+
+The `dayOfYear` read-only property gives the ordinal day of the year that the date falls on.
+This is a value between 1 and 365, or 366 in a leap year.
+
+Usage example:
+```javascript
+dt = new Temporal.DateTime(1995, 12, 7, 3, 24, 30, 0, 3, 500);
+// Print dt in ISO ordinal date format
+year = `${dt.year}`.padStart(4, '0');
+day = `${dt.dayOfYear}`.padStart(3, '0');
+hour = `${dt.hour}`.padStart(2, '0');
+minute = `${dt.minute}`.padStart(2, '0');
+second = `${dt.second}`.padStart(2, '0');
+`${year}-${day}T${hour}:${minute}:${second}`  // => 1995-341T03:24:30
+```
+
 ### datetime.**weekOfYear** : number
 
+The `weekOfYear` read-only property gives the ISO week number of the date.
+This is normally a value between 1 and 52, but in a few cases it can be 53 as well.
+ISO week 1 is the week containing the first Thursday of the year.
+For more information on ISO week numbers, see for example the Wikipedia article on [ISO week date](https://en.wikipedia.org/wiki/ISO_week_date).
+
+Usage example:
+```javascript
+dt = new Temporal.DateTime(1995, 12, 7, 3, 24, 30, 0, 3, 500);
+// Print dt in ISO week number / week day format
+year = `${dt.year}`.padStart(4, '0');
+week = `${dt.weekOfYear}`.padStart(2, '0');
+hour = `${dt.hour}`.padStart(2, '0');
+minute = `${dt.minute}`.padStart(2, '0');
+second = `${dt.second}`.padStart(2, '0');
+`${year}-W${week}-${dt.dayOfWeek}T${hour}:${minute}:${second}`  // => 1995-W49-4T03:24:30
+```
+
 ### datetime.**daysInMonth** : number
 
+The `daysInMonth` read-only property gives the number of days in the month that the date falls in.
+This is 28, 29, 30, or 31, depending on the month and whether the year is a leap year.
+
+Usage example:
+```javascript
+// Attempt to write some mnemonic poetry
+const monthsByDays = {};
+for (let month = 1; month <= 12; month++) {
+    const dt = Temporal.DateTime.from({month});
```

Comment:
Will this work, given

>  At least the `year`, `month`, and `day` properties must be present.

?

Reply:
Oops! I wrote the example before we decided that, will update.